internal/tokens/https/https.go

```diff
@@ -216,7 +216,7 @@ func (h *HTTPS) eventLoop() {
 	}
 }
 
-func (h *HTTPS) CreateTokenPool(ctx context.Context, identity *fftypes.Identity, pool *fftypes.TokenPool) error {
+func (h *HTTPS) CreateTokenPool(ctx context.Context, pool *fftypes.TokenPool) error {
 	data := createPoolData{
```

Comment:
This is mostly a note to myself, but this is where the resolved key for the token connector needs to propagate into the REST call. Opened https://github.com/hyperledger/firefly-tokens-erc1155/pull/30 to hopefully enable that to happen.